models/sale.coffee

```diff
@@ -122,7 +122,7 @@ module.exports = class Sale extends Backbone.Model
 
   # Is there a way to manually end/close out a live sale besides looking at end_at?
   isLiveOpen: ->
-    moment().isBefore(@get 'end_at') and moment().isAfter(@get 'live_start_at')
+    @get('live') != false and moment().isAfter(@get 'live_start_at')
```

Comment:
Is it possible not to compare to anything at all, and just use the boolean directly? `@get('live') and moment().isAfter(@get 'live_start_at')`